packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react';
+import { useCtraderServiceToken } from '@deriv/api';
+import { Button, Text } from '@deriv/quill-design';
+import { getPlatformFromUrl } from '../../../../../helpers/urls';
+import { THooks, TPlatforms } from '../../../../../types';
+import { AppToContentMapper, PlatformDetails, PlatformToLabelIconMapper, PlatformUrls } from '../../../constants';
+
+type TMT5TradeLinkProps = {
+    app?: keyof typeof AppToContentMapper;
+    isDemo?: THooks.ActiveAccount['is_virtual'];
+    platform?: TPlatforms.All;
+    webtraderUrl?: THooks.MT5AccountsList['webtrader_url'];
+};
+
+const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', isDemo = false, platform, webtraderUrl = '' }) => {
```

Comment:
Dont need isDemo coming as prop. Pls define it here 
const { data: activeAccount } = useActiveTradingAccount();
const isDemo = activeAccount?.is_virtual;